app/features/TransactionSlice.ts

```diff
@@ -1,4 +1,11 @@
-import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import {
+    createAsyncThunk,
+    createSlice,
+    PayloadAction,
+    isAnyOf,
+    isFulfilled,
+} from '@reduxjs/toolkit';
+import { Mutex, MutexInterface } from 'async-mutex';
```

Comment:
You are using a new dependency, but you have not added it to package.json.

Reply:
weird.. just ran `yarn add` as usual... I'll try again